examples/flutter_gallery/lib/demo/cupertino/cupertino.dart

```diff
@@ -3,8 +3,8 @@
 // found in the LICENSE file.
 
 export 'cupertino_activity_indicator_demo.dart';
+export 'cupertino_alert_demo.dart';
 export 'cupertino_buttons_demo.dart';
-export 'cupertino_dialog_demo.dart';
```

Comment:
What happened to this file?  Did you delete it?  Or just rename it?

Reply:
Ya, this is a rename it seems